wayback-webapp/src/main/webapp/WEB-INF/ComplexAccessPoint.xml

```diff
@@ -154,23 +154,26 @@
 -->
 <!--
   <bean name="netsecureaccesspoint" parent="standardaccesspoint">
-    <property name="accessPointPath" value="http://localhost:8080/netsecure/"/>
+    <property name="accessPointPath" value="${wayback.url.prefix}"/>
```

Comment:
I am going to add back the `/netsecure/` path here and merge. Thanks @chasehd for keeping with this and getting #260 fixed!